cli/monitoring/custom-monitoring.yaml

```diff
@@ -14,18 +14,16 @@ create_monitor:
       component_id: azureml:my_custom_signal:1.0.0
       input_datasets:
         production_data:
-          input_dataset:
+          input_data:
             type: uri_folder
             path: azureml:custom_without_drift:1
-          dataset_context: test
+          data_context: test
           pre_processing_component: azureml:custom_preprocessor:1.0.0
       metric_thresholds:
-        - metric_name: std_deviation
-          threshold: 2
+        numerical:
```

Comment:
we don't do numerical/categorical as top level metric differentiator for custom signals, this should be metric name and the associated threshold